xyz

```diff
@@ -102,7 +102,7 @@ inc() {
           for (( idx = ${#xs[@]} - 1 ; idx > 0 ; idx -= 1 )) ; do
             pattern="^($number)$"
             if [[ ${xs[idx]} =~ $pattern ]] ; then
-              parts=(${xs[@]})
+              parts=("${xs[@]}")
```

Comment:
```bash
              parts=(${xs[@]})
                     ^-- SC2206
```

[SC2206][1]: Quote to prevent word splitting, or split robustly with `mapfile` or `read -a`.


[1]: https://github.com/koalaman/shellcheck/wiki/SC2206